travis.cfg

```diff
@@ -1,10 +1,16 @@
 [buildout]
 extends =
     https://raw.github.com/collective/buildout.plonetest/master/travis-4.3.x.cfg
+    https://raw.github.com/collective/buildout.plonetest/master/qa.cfg
     buildout.d/base.cfg
 parts = download install test createcoverage code-analysis
+versions = versions
 
 [code-analysis]
 recipe = plone.recipe.codeanalysis
 return-status-codes = True
 
+[versions]
+setuptools = 8.3
+plone.recipe.plone = 4.3.6
+plone.app.contenttype = 1.1b4
```

Comment:
@prakharjoshi this is a typo, the package name is plone.app.contenttypes. Though, it shouldn't be necessary to pin versions specifically for travis.